webxrlayers-1.bs

```diff
@@ -797,7 +807,7 @@ When this method is invoked, the user agent MUST run the following steps:
   1. Initialize |layer|'s [=XRCompositionLayer/session=] to |session|.
   1. Let |layer|'s {{XRCubeLayer/space}} be the {{session}}'s [=viewer reference space=].
   1. Let |layer|'s {{XRCubeLayer/orientation}} be a new {{DOMPoint}}.
-  1. let |textureType| be {{XRTextureType/"texture-array"}}.
+  1. let |textureType| be {{XRTextureType/"cubemap-texture"}}.
```

Comment:
We do not have textureType exposed to user, right?

Reply:
No, I just double checked that myself. We shouldn't have this as a publically accessible value, given that it's never actually going to be exposed by the API and it's use is implied by the layer type. The DOMPointReadOnly change is good, though.